app/models/service.rb

```diff
@@ -0,0 +1,52 @@
+class Service < ActiveRecord::Base
```

Comment:
This looks like the same class in @andela-amagana PR. Did you merge his work here and will this result in conflict later?